### PR TITLE
OCPBUGS#27254: Remove API Server and Ingress sections

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.12

Issue: [OCPBUGS-27254](https://issues.redhat.com/browse/OCPBUGS-27254)

Link to docs preview: https://70394--ocpdocs-pr.netlify.app/openshift-enterprise/latest/security/cert_manager_operator/cert-manager-creating-certificate

QE review:
- [x] QE has approved this change. @lunarwhite

### CODE REVIEW

File: security/cert_manager_operator/cert-manager-creating-certificate.adoc [hunk 1]
```diff
@@ -10,19 +10,11 @@ By using the {cert-manager-operator}, you can manage certificates, handling task
 
 include::modules/cert-manager-certificate-mgmt.adoc[leveloffset=+1]
 
-include::modules/cert-manager-certificate-api-server.adoc[leveloffset=+1]
-
-include::modules/cert-manager-certificate-ingress.adoc[leveloffset=+1]
-
```
Comment (lunarwhite) [inline security/cert_manager_operator/cert-manager-creating-certificate.adoc:16]: Do we also need to remove these two files together？

https://github.com/openshift/openshift-docs/blob/enterprise-4.12/modules/cert-manager-certificate-api-server.adoc

https://github.com/openshift/openshift-docs/blob/enterprise-4.12/modules/cert-manager-certificate-ingress.adoc

**Review by lunarwhite — COMMENTED** [review]
otherwise LGTM

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**